src/App.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import HeaderApp from './components/Header/HeaderApp';
```

Comment:
Why `HeaderApp` and not just `Header`?

Reply:
Because I use name Header for styles. I'm going to change it